lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -312,7 +312,11 @@ def increment_positions_on_lower_items(position, avoid_id = nil)
             scope = scope.where("#{quoted_table_name}.#{self.class.primary_key} != ?", avoid_id)
           end
 
-          scope.where("#{quoted_position_column_with_table_name} >= ?", position).increment_all
+          if ::ActiveRecord::Base.connection_config[:adapter].match? /sqlite/
```

Comment:
Could you let me know why you're targeting only `sqlite`? :)

Reply:
The fix didn't work with sqlite, probably because it ignores the ordering. And using the sequentially incrementation for MySQL and Psql seemed to be too resourcefulness to me :)